lib/perl/Genome/InstrumentData/AlignmentResult_regenerate.t

```diff
@@ -235,4 +235,23 @@ sub get_test_alignment_results {
     return ($ar1, $ar2, $merged_result, $smaller_merged_result, $bad_merged_result);
 }
 
+subtest 'test per lane bam removal and recreation - AlignedBamResult accessors' => sub {
+    my $temp_allocation_dir = Genome::Sys->create_temp_directory();
+    my $owner = Genome::Sys::User->get(username=>"apipe-tester");
+
+    my $temp_allocation = $allocation_class->generate_obj(
+        mount_path => $temp_allocation_dir,
+        owner => $owner,
+    );
+
+    # The revivified bam will be in a different location
+    my $new_path = $ar2->revivified_alignment_bam_file_paths(disk_allocation => $temp_allocation);
+    # This test currently fails because the bam_path points to the original
+    # output_dir. This will change once we have a universal method to get to
+    # the bam's path (which shouldn't be called bam_path)
+    # is($ar2->bam_path, File::Spec->join($temp_allocation_dir, $per_lane_bam), 'bam_path correct after revivification');
```

Comment:
It might be better to have this single test [skip](http://search.cpan.org/~exodist/Test-Simple/lib/Test/More.pm#Conditional_tests) instead of being commented out.